examples/tabular/example_tabular_predictor_v2.py

```diff
@@ -0,0 +1,62 @@
+
+from autogluon.tabular import TabularPrediction as task
+from autogluon.tabular.task.tabular_prediction.predictor import TabularPredictor
+from autogluon.tabular.task.tabular_prediction.predictor_v2 import TabularPredictorV2
+
+
+################
+# Loading data #
+################
+
+train_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')
+test_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/test.csv')
+label = 'class'
+hyperparameters = {'RF': {}}
+train_data = train_data.head(1000)  # subsample for faster demo
+
+##################################
+# Fitting with the old Predictor #
+##################################
+
+predictor1 = task.fit(
+    train_data=train_data,
+    label=label,
+    hyperparameters=hyperparameters,
+)
+predictor1.leaderboard(test_data)
+
+##################################
+# Fitting with the new Predictor #
+##################################
+
+predictor2 = TabularPredictorV2(label=label)
```

Comment:
how we decide which args to the Predictor, which to the fit. `label` is related to `train_data`. If we change the data later, we may change `label` as well

Reply:
Will plan to address this question once pre model-fit methods are added such as fit_feature_generator so the utility of the placement of such arguments can be demonstrated and reviewed.